packages/flutter_tools/lib/src/macos/native_assets.dart

```diff
@@ -39,19 +39,19 @@ Future<Iterable<Asset>> dryRunNativeAssetsMacOSInternal(
   bool flutterTester,
   NativeAssetsBuildRunner buildRunner,
 ) async {
-  const OS targetOs = OS.macOS;
-  final Uri buildUri_ = nativeAssetsBuildUri(projectUri, targetOs);
+  const OS targetOS = OS.macOS;
+  final Uri buildUri_ = nativeAssetsBuildUri(projectUri, targetOS);
```

Comment:
Just curious, why the `_` suffix here?

Reply:
`nativeAssetsBuildUri` used to be called `buildUri` in a previous version. Cleaned up.